scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -578,6 +623,9 @@ def add_testsuites(self, testsuite_filter=None):
                         else:
                             self.testsuites[suite.name] = suite
 
+                    if scenario_count > 5 and self.options.log_level == "QA":
```

Comment:
`and self.options.log_level == "QA"` looks redundant,

Reply:
those will be cleaned up, I was getting the wrong level at some point and did not want the messages to interfere.